Sources/ToucanContent/Markdown/MarkdownBlockDirective.swift

```diff
@@ -63,3 +63,36 @@ public struct MarkdownBlockDirective: Codable {
         self.output = output
     }
 }
+
+extension MarkdownBlockDirective.Parameter: Equatable {
+    public static func == (
```

Comment:
No need to actually implement these functions, just conform everything to Equatable and the compiler will do the job. ;)